rbi/stripe/services/capital/financing_transaction_service.rbi

```diff
@@ -0,0 +1,68 @@
+# File generated from our OpenAPI spec
+# frozen_string_literal: true
+
+# typed: true
+module Stripe
+  module Capital
+    class FinancingTransactionService < StripeService
+      class ListParams < Stripe::RequestParams
+        # For transactions of type `paydown` and reason `automatic_withholding` only, only returns transactions that were created as a result of this charge.
+        sig { returns(String) }
+        attr_accessor :charge
+        # A cursor for use in pagination. `ending_before` is an object ID that defines your place in the list. For instance, if you make a list request and receive 100 objects, starting with `obj_bar`, your subsequent call can include `ending_before=obj_bar` in order to fetch the previous page of the list.
+        sig { returns(String) }
+        attr_accessor :ending_before
+        # Specifies which fields in the response should be expanded.
+        sig { returns(T::Array[String]) }
+        attr_accessor :expand
+        # Returns transactions that were created that apply to this financing offer ID.
```

Comment:
I'm not sure anyone will actually be reading these files, but if they are, some line breaks between items would go a long way! e.g.

```rb
        # A limit on the number of objects to be returned. Limit can range between 1 and 100, and the default is 10.
        sig { returns(Integer) }
        attr_accessor :limit

        # Only returns transactions that are responsible for reversing this financing transaction ID.
        sig { returns(String) }
        attr_accessor :reversed_transaction
```